load_balancing.go

```diff
@@ -47,6 +47,9 @@ type LoadBalancerOrigin struct {
 	// weight is used to scale the origin's open connections.
 	Weight float64             `json:"weight"`
 	Header map[string][]string `json:"header"`
+	// The virtual network subnet ID the origin belongs in.
+	// Virtual network must also belong to the account.
+	VirtualNetworkId string `json:"virtual_network_id,omitempty"`
```

Comment:
```suggestion
	VirtualNetworkID string `json:"virtual_network_id,omitempty"`
```

Reply:
i attempted to automatically apply this for you but because your using the default branch (`master`), the branch protections won't allow it to be applied.

Reply:
Thanks for the heads up! Changing it now